.env_sample

```diff
@@ -0,0 +1,3 @@
+GOOGLE_ANALYTICS_TRACKING_ID=GA1
```

Comment:
yes, without an .env file there will be some errors in the console, especially ga errors.
<img width="1274" alt="screen shot 2017-04-10 at 16 15 42" src="https://cloud.githubusercontent.com/assets/1902266/24868796/19a9c4cc-1e09-11e7-89de-1c9c0e6510f4.png">

Do you want me to update the README?